docs/catalyst_101.md

```diff
@@ -280,37 +276,40 @@ All meetings have pre-designated facilitators and note-takers that are specified
 - After running through the action items, we open the floor to discuss anything else relevant to the whole team. If you want to talk about something specific, add it to something to the agenda in the PUDL Huddle document under the appropriately titled meeting header (Ex: Weekly Meeting YYYY-MM-DD). If there isn’t one yet, feel free to make one. Sometimes there are no agenda items in which case we’ll end the meeting early.
 
-(business-team-meetings)=
+(work-team-meetings)=
 
-#### Business Team Meetings - Team Specific (time and occurrence differs)
+#### Weekly Sprint Meetings - Team Specific
 
-**Attendees**: Team members
-**Notes:** See notes documents for specific teams: [Business Teams](https://drive.google.com/open?id=1KuF1sJGQMWmODpD5LhnWVN61Ymbmg9xR)
+**Attendees**: Team members.
+**Notes:** See notes documents for specific teams: [work teams](https://drive.google.com/drive/folders/1CHbOJU6W3r3DE2LEQqbcfdme5DK-qufz)
 
-- Catalyst organizes internal tasks into a series of teams that meet on a semi-regular basis (defined in the [Teams Index](https://docs.google.com/document/u/0/d/1B9wzU3IuNlJbqG-4XpcpJnIpXhq8IRzf6m4lJnxXVvo/edit)) to discuss and execute issues related to their domain. Refer to the Teams Index for a list of the current teams, their responsibilities, and how often they meet.
+- Work teams usually meet twice a sprint (once a week). First to plan the sprint and second to check-in half way through. See
+  [above](sprint-planning) for more detail on these meetings.
+- These meetings will consult the GitHub boards specific to each team.
 
-- Teams are not mandatory, but contractors are encouraged and members are expected to participate in at least one.
+(internal-team-meetings)=
 
-(quarterly-advisory-committee-meeting-1-hours)=
+#### Internal Team Meetings - Team Specific (time and occurrence differs)
 
-#### Quarterly Advisory Committee Meeting (1+ hours)
+**Attendees**: Team members.
+**Notes:** See notes documents for specific teams: [Internal teams](https://drive.google.com/open?id=1KuF1sJGQMWmODpD5LhnWVN61Ymbmg9xR)
 
-**Attendees**: Contractors & Members
-**Notes:** [Advisory Committee](https://drive.google.com/open?id=1f7y2csPj4rtJJoYJW05kwHyY7-sD_5Fb)
+- Catalyst organizes internal tasks into a series of teams that meet on a semi-regular basis to discuss and execute issues related to their domain. See [above](internal-teams) for a breif description, or refer to the [Teams Index](https://docs.google.com/document/u/0/d/1B9wzU3IuNlJbqG-4XpcpJnIpXhq8IRzf6m4lJnxXVvo/edit) for a list of current teams, their responsibilities, and how often they meet.
+- Teams are not mandatory, but contractors are encouraged and members are expected to participate in at least one.
```

Comment:
Would we approve a member candidate who refused to be in any internal teams? Would we be happy with a member who refused to be in any internal teams? From my perspective it seems like we wouldn't... but maybe that's a bigger discussion.

Reply:
I think we would definitely let people opt out of teams if we got large enough. Even now, if someone just wanted to code I think that would be fine--we have been talking about trying to reduce team size anyways.